### ISSUE TITLE
Xemu / Could not mount drive G

### ISSUE THREAD

**Issue by B4rabbas** [dracc/NevolutionX#61]
Hi,
NevolutionX is freshly installed as my default dashboard.
Installed from FTP an renamed default.xbe into xboxdash.xbe.

At startup I got this message :
 ```
Mounting warning: Could not mount drive G
EEPROM MacAdress = 00 50 f2 09 9d 4e
Transceiver link state : 
Running at 100 Mbps in full duplex mode
Network interruption initialized successfully
```
And unable (obviously) to launch DVD, give the same error message.

**Comment by GXTX** [comment]
That's to be expected, a normal sized drive is smaller than what a drive large enough to have a 7th partition would need to be.

**Comment by dracc** [comment]
Closed by #48a28c6
